src/CodeGeneratorGo.cs

```diff
@@ -79,21 +71,55 @@ public override string ImplementationFileExtension
                     methodGroup.Name += "group";
                 }
                 ReservedFiles.Add(methodGroup.Name);
-                var methodGroupTemplate = new MethodGroupTemplate
-                {
-                    Model = methodGroup
-                };
-                await Write(methodGroupTemplate, FormatFileName(methodGroup.Name).ToLowerInvariant());
+                var methodGroupTemplate = new MethodGroupTemplate { Model = methodGroup };
+                await Write(methodGroupTemplate, FormatFileName(methodGroup.Name));

```

Comment:
To lower is still a good idea, to avoid git issues

Reply:
Oh, it is other place, nvm

Reply:
You are right, will fix.

Reply:
Hah so it is.  :)